org.eclipse.xtext.ide.tests/testlang-src/org/eclipse/xtext/ide/tests/testlanguage/scoping/TestLanguageScopeProvider.java

```diff
@@ -14,6 +14,6 @@
  * See https://www.eclipse.org/Xtext/documentation/303_runtime_concepts.html#scoping
  * on how and when to use it.
  */
-class RenameTestLanguageScopeProvider extends AbstractRenameTestLanguageScopeProvider {
+public class TestLanguageScopeProvider extends AbstractTestLanguageScopeProvider {
```

Comment:
Although this diff looks a little bit weird, I think it should be fine.

Reply:
please report a bug to github ;)